src/app/zap-templates/zcl/air-pressure-measurement.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<configurator>
+  <domain name="CHIP"/>
+  <cluster>
+    <domain>Measurement &amp; Sensing</domain>
+    <name>Air Pressure Measurement</name>
+    <code>0x0407</code>
+    <define>AIR_PRESSURE_MEASUREMENT_CLUSTER</define>
+    <client init="false" tick="false">true</client>
+    <server init="false" tick="false">true</server>
+    <description>Air pressure measurement cluster in hPA with 1 digit fixed point resolution</description>
+    <attribute side="server" code="0x0000" define="AIR_PRESSURE_MEASUREMENT_MEASURED_VALUE" type="INT16U" min="8700" max="11000" writable="false" default="10132" optional="false">measured value</attribute>
```

Comment:
The 8700 is suspiciously close to "the lowest we've measured in a hurricane recently".  Is that really what the min should be in the spec?  Do we ever expect one of these sensors to be in a tornado?

11000 seems to assume this will never be used in any sort of submersible?

Reply:
Are you suggesting to remove the min / max values and just use the value range of the INT16U as min / max values? The min / max values where intended to indicate the possible value range of the atmospheric pressure. I would agree that maybe changing them a bit lower / higher would be beneficial to compensate some sensor measurement tolerances. So I would e.g. suggest 8000 for min and 12000 for max. 

The intention of this cluster was to measure air pressure / atmospheric pressure. I don't know how this could be related to submersible applications? Could you provide an example.